gradle.properties

```diff
@@ -1,5 +1,5 @@
 projectVersion=1.3.2
 kotlinVersion=1.6.10
 exposedVersion=0.37.3
-lsp4jVersion=0.14.0
+lsp4jVersion=0.15.0
```

Comment:
Any API-breaking changes since 0.14? Otherwise I'd say this is good to go.

Reply:
No API breaking changes in this release, mainly extending support for LSP 3.17.0